app/views/physical_infra_topology/show.html.haml

```diff
@@ -1,39 +1 @@
-- tooltipOptions = {"tooltip-placement" => "bottom", "uib-tooltip" => "{{vm.legendTooltip}}"}
-.topology{'ng-controller' => "physicalInfraTopologyController as vm"}
-  .legend
-    %label#selected
-    %div{'ng-if' => "vm.kinds"}
-      %kubernetes-topology-icon{tooltipOptions, :kind => "PhysicalRack"}
-        %label
-          %i.pficon.pficon-enterprise
-          = _("Physical Racks")
-      %kubernetes-topology-icon{tooltipOptions, :kind => "PhysicalServer"}
-        %label
-          %i.pficon.pficon-server-group
-          = _("Physical Servers")
-      %kubernetes-topology-icon{tooltipOptions, :kind => "PhysicalSwitch"}
-        %label
-          %i.ff.ff-network-switch
-          = _("Physical Switches")
-      %kubernetes-topology-icon{tooltipOptions, :kind => "Host"}
-        %label
-          %i.pficon.pficon-screen
-          = _("Hosts")
-      %kubernetes-topology-icon{tooltipOptions, :kind => "Vm"}
-        %label
-          %i.pficon.pficon-screen
-          = _("VMs")
-
-  .alert.alert-info.alert-dismissable
-    %button.close{"aria-hidden" => "true", "data-dismiss" => "alert", :type => "button"}
-      %span.pficon.pficon-close
-    %span.pficon.pficon-info
-    %strong
-      = _("Click on the legend to show/hide entities, and double click/right click the entities in the graph to navigate to their summary pages.")
-
-  = render :partial => "shared/topology/not_found_vm"
-
-  %kubernetes-topology-graph{:items => "vm.items", :relations => "vm.relations", :kinds => "vm.kinds"}
-
-:javascript
-  miq_bootstrap('.topology');
+= render :partial => "ems_physical_infra/show_topology"
```

Comment:
Why introducing this inconsistency? All the other topology views are using `*_topology/show.html.haml` file structure...